tests/Properties.hs

```diff
@@ -1212,6 +1212,14 @@ prop_elemIndexEnd1LL c xs = (L.elemIndexEnd c (L.pack xs)) ==
 prop_elemIndexEnd2LL c xs = (L.elemIndexEnd c (L.pack xs)) ==
                            ((-) (fromIntegral (length xs) - 1) `fmap` L.elemIndex c (L.pack $ reverse xs))
 
+prop_elemIndexEnd1LC c xs = (LC.elemIndexEnd c (LC.pack xs)) ==
+                           (case LC.elemIndex c (LC.pack (reverse xs)) of
+                                 Nothing -> Nothing
+                                 Just i  -> Just (fromIntegral (length xs) -1 -i))
+
+prop_elemIndexEnd2LC c xs = (LC.elemIndexEnd c (LC.pack xs)) ==
+                           ((-) (fromIntegral (length xs) - 1) `fmap` LC.elemIndex c (LC.pack $ reverse xs))
```

Comment:
What's the difference between `prop_elemIndexEnd1LC` and `prop_elemIndexEnd2LC`? They look equivalent to me.

Reply:
I saw a duplicate test elsewhere (look for `prop_elemIndexEnd1LL` and `prop_elemIndexEnd2LL`. I can remove the duplicates in both places if you'd like.

Reply:
Ah, I see. Yes, please remove other duplicates as well.